src/write_prometheus_test.c

```diff
@@ -439,11 +439,154 @@ DEF_TEST(target_info) {
   return 0;
 }
 
+void format_text(strbuf_t *buf);
+int prom_write(metric_family_t const *fam, user_data_t *ud);
+int alloc_metrics(void);
+void free_metrics(void);
```

Comment:
Unnecessary if test `#include`s the tested plugin code.

Reply:
✅ done